src/simplewallet/simplewallet.cpp

```diff
@@ -4763,9 +4763,14 @@ bool simple_wallet::try_connect_to_daemon(bool silent, uint32_t* version)
   if (!m_wallet->check_connection(version))
   {
     if (!silent)
-      fail_msg_writer() << tr("wallet failed to connect to daemon: ") << m_wallet->get_daemon_address() << ". " <<
-        tr("Daemon either is not started or wrong port was passed. "
-        "Please make sure daemon is running or change the daemon address using the 'set_daemon' command.");
+    {
+      if (m_wallet->is_offline())
+        fail_msg_writer() << tr("wallet failed to connect to daemon, because it is set to offline mode");
```

Comment:
Wouldn't this still show the error message, even when there is no error because the user has explicitly requested `--offline`?

Reply:
This is inside "if (!m_wallet->check_connection(version))" (ie, the "falied" branch).

Reply:
Sorry, I am not following...
Are you saying _"Error: wallet failed to connect to daemon, because it is set to offline mode"_ will not be displayed if user specifies ` --offline`?

Reply:
No, the opposite :)

Reply:
Then... it seems it does the _opposite_ of fixing #7013 ;-)

Can we change the message to "Wallet is running in offline mode." and make it purely informational (rather than showing up as an error)?

Reply:
Well, it doesn't have a daemon so it can't do whatever you asked it to do, right ? Am I just missing your point ?

Reply:
I was seeing this "error" immediately upon opening the wallet using the `--offline` option.  In my case at least, it was not an error, but deliberate.  Maybe that situation is already fixed by preventing `m_wallet->auto_refresh()` below?

Reply:
It doesn't try to auto refresh now when --offline is set. It *should* display that error only when you try to use the daemon with things like.... manual refresh  I guess. Or start_mining, etc.

Reply:
Okay, let's see how this goes and I can open another issue if the "error" is still showing up in the wrong circumstances.
Thank you.